# Generated by Django 2.1.3 on 2018-11-24 21:34

from django.conf import settings
import django.core.validators
from django.db import migrations, models
import django.utils.timezone


class Migration(migrations.Migration):

    initial = True

    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
    ]

    operations = [
        migrations.CreateModel(
            name='Transfer',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('recipient_name', models.CharField(max_length=120)),
                ('recipient_account', models.CharField(max_length=26, validators=[django.core.validators.RegexValidator(message='Niewłaściwy numer konta', regex='[0-9]{26}')])),
                ('title', models.CharField(max_length=120)),
                ('amount', models.DecimalField(decimal_places=2, max_digits=10)),
                ('date', models.DateTimeField(default=django.utils.timezone.now)),
                ('sender', models.ForeignKey(editable=False, on_delete=None, to=settings.AUTH_USER_MODEL)),
            ],
        ),
    ]
